app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<!--each one is a statement not a hidden visible ** just make it a list-->
+
+    <h2>TASKS</h2>
+      <!-- <p><%#= link_to "Edit", edit_task_path(task.id) %></p> -->
+      <!--<p><%#= link_to "Delete", delete_task_path(task.id) %></p> -->
+    <!-- <ol> -->
+      <% @tasks.each_with_index do |task, i|  %>
+        <section>
+          <h3><%= "#{i + 1}.    " %><%= link_to task.title, task_path(task.id), { :class => (task.complete) ? "completed" : ""} %></h3>
+
+          <% if task.complete == true %>
```

Comment:
The `== true` is redundant here because `if` statements are automatically checking for [truthiness](https://learn.co/lessons/truthiness-in-ruby-readme), and booleans such as `task.complete` have "truthiness" that matches their value of `true`/`false`.